### ISSUE TITLE
fetchzip can be used directly

### ISSUE THREAD

**Issue by layus** [ryantm/nixpkgs-update#90]
For this issue

```
2018-11-10T05:23:13 terminus-font-ttf 4.40.1 -> 4.46.0
2018-11-10T05:23:21 nix eval failed for pkgs.terminus_font_ttf.src.urls
2018-11-10T05:23:21 FAIL
2018-11-10T05:23:21 terminus-font-ttf 4.40.1 -> 4.46
2018-11-10T05:23:29 nix eval failed for pkgs.terminus_font_ttf.src.urls
2018-11-10T05:23:29 FAIL
```

it appears that the fonts are produced by a `fetchzip` invocation

```nix
{ stdenv, fetchzip }:

let
  version = "4.40.1";
in fetchzip rec {
  name = "terminus-font-ttf-${version}";

  url = "http://files.ax86.net/terminus-ttf/files/${version}/terminus-ttf-${version}.zip";

  postFetch = ''
    unzip -j $downloadedFile

    for i in *.ttf; do
      local destname="$(echo "$i" | sed -E 's|-[[:digit:].]+\.ttf$|.ttf|')"
      install -Dm 644 "$i" "$out/share/fonts/truetype/$destname"
    done

    install -Dm 644 COPYING "$out/share/doc/terminus-font-ttf/COPYING"
  '';

  sha256 = "0cfkpgixdz47y94s9j26pm7n4hvad23vb2q4315kgahl4294zcpg";

  meta = with stdenv.lib; {
```

There are many ways to work around this. Ignoring evaluation failure may work, as well as adding support for `$pkg.url` directly.

**Comment by layus** [comment]
> [<img alt="@ryantm" width="16" height="16" src="https://avatars1.githubusercontent.com/u/4804?s=60&amp;v=4">](/ryantm) [ryantm](/ryantm) (Ryan Mulligan) closed this in `[9418693](/ryantm/nixpkgs-update/commit/94186939d8e11a9b6b3e476cd27e2eb6d32ea7d7)` [32 minutes ago](#event-1973321324)

:1st_place_medal: That's a _quick_ fix !

**Comment by ryantm** [comment]
Woo. It is fixed by checking `$pkg.url` if `$pkg.src.url` fails! Thanks for the help.
